websites/C/ChatGPT/presence.ts

```diff
@@ -0,0 +1,48 @@
+const presence = new Presence({ clientId: "1102935778570547282" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/qw4f6EN.png",
+	Talking = "https://i.imgur.com/hSv055V.png",
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname } = document.location,
+		presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		showTitle = await presence.getSetting<boolean>("showTitle"),
+		isTalking = document.querySelector(
+			'[class*="text-2xl"] > span:nth-child(3)'
+		);
+
+	let words = 0;
+	for (const element of document.querySelectorAll(
+		'[class*="flex flex-col items-center"] > *'
+	)) {
+		const text = element.textContent.replace(
+			/([.])|(\n)|([0-9]*)|(\/)|()/gm,
+			""
+		);
```

Comment:
Otherwise it gives an incorrect number due to not splittings words like (this is due to how you get the textContent)
```yaml
Hi,How are you?
```
to 
```
Hi, How
```
```suggestion
		const text = element.textContent.replace(/(, )|(,\n)|(,)|([.] )|([.])/gm, " ").replace(
			/([0-9]*)|(\/)|(')|(,)|(  )/gm,
			""
		);
```